src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -167,7 +167,18 @@ private static async Task<WorkerLanguageVersionInfo> GetVersion(string pythonExe
             return null;
         }
 
-        public static async Task<string> VerifyVersion(string pythonExe = "python")
+        public static void SetWorkerRuntimeVersionPython(WorkerLanguageVersionInfo version)
+        {
+            if (version == null)
+            {
+                return;
```

Comment:
Do we want to throw an exception when worker version cannot be resolved? This is an issue if the customer is not configuring their PATH properly.

Reply:
Technically this will never happen, as `ValidatePythonVersion` will throw an exception before it gets here. But I will add ArgumentNullException as a safeguard. Thanks